spec/models/order_spec.rb

```diff
@@ -1,5 +1,20 @@
 require "rails_helper"
+require "support/factory_girl"
 
 RSpec.describe Order, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  it 'is valid' do
+    order = create(:order)
+    expect(order).to be_valid
+  end
+
+  it 'is invalid without a user_id' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.